lib/views/pay/pay_view.dart

```diff
@@ -120,6 +148,23 @@ class _PayViewState extends State<PayView> {
     return Scaffold(
       appBar: AppBar(
         elevation: 0,
+        actions: [
+          IconButton(
+            icon: const ImageIcon(AssetImage('assets/images/scanner.png')),
+            onPressed: () async {
+              if (Platform.isAndroid || Platform.isIOS) {
```

Comment:
```suggestion
              if (Platform.isAndroid || Platform.isIOS) {
```

no use this to disable the button